app/controllers/concerns/audited_reports/process_control_stats.rb

```diff
@@ -0,0 +1,360 @@
+module AuditedReports::ProcessControlStats
+  include Reports::Pdf
+  include Reports::Period
+  include Parameters::Risk
+
+  def process_control_stats
+    @controller  = params[:controller_name]
+    final        = params[:final] == 'true'
+    @title       = t("#{@controller}.process_control_stats_title")
+    @from_date   = 1.year.ago.to_date
+    @to_date     = Time.zone.now.to_date
+    @risk_levels = []
+    @filters     = []
+
+    @columns = [
+      ['process_control', BestPractice.human_attribute_name('process_controls.name'), 60],
+      ['effectiveness', t("#{@controller}.process_control_stats.average_effectiveness"), 20],
+      ['weaknesses_count', t('review.weaknesses_count'), 20]
+    ]
+
+    conclusion_reviews = ConclusionFinalReview.list_all_by_date(
+      @from_date, @to_date
+    ).scored_for_report
+
+    user_review        = Current.user.reviews.list_with_final_review.last
+    business_unit_type = user_review ? user_review.business_unit_type : nil
```

Comment:
```suggestion
    business_unit_type = user_review&.business_unit_type
```
Para eso es el safe path.